src/Microsoft.OData.Client/EntityTracker.cs

```diff
@@ -540,24 +540,24 @@ private void EnsureIdentityToResource()
         {
             if (null == this.identityToDescriptor)
             {
-            #if PORTABLELIB && WINDOWSPHONE
+#if PORTABLELIB && WINDOWSPHONE

```

Comment:
Anything else other than indentation changed in this PR? No logic changes, correct?

Reply:
Won't change. 

There's a build warning complaining about this. Why do we wait to another PR that maybe never happen?

Reply:
I don't see any related build warning in the currently merged build.
https://identitydivision.visualstudio.com/OData/_build/results?buildId=177022

If this is one of the warnings show above, we can definitely fix it.

Reply:
I think Biao was just confirming that there were no material changes.  I am fine with fixing indentation issues that we find along the way.

---
In reply to: [214475994](https://github.com/OData/odata.net/pull/1242#discussion_r214475994) [](ancestors = 214475994)

Reply:
I'm fine with fixing indentation as part of this PR.

---
In reply to: [214489682](https://github.com/OData/odata.net/pull/1242#discussion_r214489682) [](ancestors = 214489682)